website/screens/components/date-input/code/examples/errorHandling.ts

```diff
@@ -0,0 +1,39 @@
+import { DxcDateInput, DxcInset } from "@dxc-technology/halstack-react";
+import { useState } from "react";
+
+const code = `() => {
+  const [value, setValue] = useState("");
+  const [error, setError] = useState();
+  const onChange = ({ value }) => {
+    setValue(value);
+  };
+  const onBlur = ({ value, error }) => {
+    setValue(value);
+    setError(error);
+  };
+
+  return (
+    <DxcInset space="large">
+      <DxcDateInput
+        label="Formatted"
+        helperText="If the typed date doesn't match the defined format, an error will be displayed"
```

Comment:
We can add this helper text here as a regular text for the example, to give the user an explanation of how uncontrolled components work.